jenkins/tenant/aws/ecoeng_03/PolicyJenkinsfileDaily

```diff
@@ -23,7 +23,7 @@ pipeline {
         // Find the all available policies: https://github.com/redhat-performance/cloud-governance/tree/main/cloud_governance/policy
         // By default, all policies are running in dry_run="yes" mode and the whole list can be found in run_policies.py
         // POLICIES_IN_ACTION: Policies that run in the dry_run="no" mode
-        POLICIES_IN_ACTION = []
+        POLICIES_IN_ACTION = "[]"
```

Comment:
Why did u add double quotes and not single quote ?

Reply:
We can use single/ double quotes. fixed now